webpack/envs/serverConfig.js

```diff
@@ -57,6 +58,16 @@ export const serverConfig = {
       },
     ],
   },
+  optimization: {
+    minimize: !env.webpackDebug,
+    minimizer: [
+      new TerserPlugin({
+        cache: false,
+        parallel: true,
+        sourceMap: true, // Must be set to true if using source-maps in production
+      }),
+    ],
+  },
```

Comment:
Wow, were we not minifiying server code?

Reply:
We are, minification is enabled when `mode: 'production'`, however sourceMap is `false` by default. This is more to keep the options consistent for server and asset builds.

Reply:
Ah yes, `production` mode... whew.